apps/frontend/src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,51 @@
+import clsx from 'clsx';
+
+export interface HeadingProps {
+  level: 'h1' | 'h2' | 'h3' | 'h4' | 'h5';
+  children: React.ReactNode;
```

Comment:
The property `children` refers multiple elements but it can be defined as `text` and the type can be string.

Reply:
But I'm thinking that if we need to add, for example, a `span` tag around some of the text for whatever reason, this can be done. Plus it simulates the original heading and paragraph tags 
```
<Heading level="h1">Some <span className="color-red">text</span></Heading>
```